features/bdo.yml

```diff
@@ -0,0 +1,6 @@
+name: <bdo>
+description: The `<bdo>` element overrides the current directionality of text, so that the text within is rendered in a different direction.
```

Comment:
```suggestion
description: The `<bdo>` element overrides the directionality of text, so that the text within is rendered in a different direction than the surrounding text.
```